airbyte-integrations/src/main/java/io/airbyte/integrations/Integrations.java

```diff
@@ -47,6 +47,8 @@
       UUID.fromString("8442ee76-cc1d-419a-bd8b-859a090366d4"),
       new IntegrationMapping("airbyte/integration-singer-csv-destination", "0.1.0"));
 
+  private static final String INTEGRATION_OVERRIDE_VERSION = "INTEGRATION_OVERRIDE_VERSION";
```

Comment:
I would call it: `USE_DEV_INTEGRATIONS` and make it a boolean instead since it doesn't really make sense to set an override version that is not `dev`